test/runtests.jl

```diff
@@ -10,12 +10,11 @@ using StaticArrays
 using SparseArrays
 using PlyIO
 using Unitful
-using Test, Random, Plots
+using MeshViz
+import CairoMakie as Mke
```

Comment:
Please move this import to a separate paragraph like it is done in MeshViz.jl tests. This is to highlight that we can switch backends if necessary.